dist/@types/componentRelay.d.ts

```diff
@@ -204,9 +204,10 @@ export default class ComponentRelay {
     /**
      * Gets the Item's appData value for the specified key.
      * Uses the default domain (org.standardnotes.sn).
+     * This function is used with Items returned from streamContextItem() and streamItems()
      * @param item The Item to get the appData value from.
      * @param key The key to get the value from.
      */
-    getItemAppDataValue(item: SNItem, key: AppDataField): any;
+    getItemAppDataValue(item: any, key: AppDataField): any;
```

Comment:
Could it not be a partial of SNItem?
```suggestion
    getItemAppDataValue(item: Partial<SNItem>, key: AppDataField): any;
```

Reply:
Mmm... `appData` doesn't exist on `SNItem`. Every note item sent to the `componentRelay` is handled by [jsonItem](https://github.com/standardnotes/snjs/blob/9b47fa330d29693e4534ced6b61b1cded115d707/packages/snjs/lib/services/component_manager.ts#L551). So it's just a plain object.

Reply:
Just realized that this should be of type `ItemMessagePayload` instead.